<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PubDialog</class>
 <widget class="QDialog" name="PubDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>343</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>518</width>
    <height>378</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Whisper Publisher</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="wShapeLabel">
           <property name="font">
            <font>
             <family>Sans Serif</family>
             <pointsize>9</pointsize>
             <stylestrategy>PreferAntialias</stylestrategy>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Business Type</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Sans Serif</family>
             <pointsize>9</pointsize>
             <stylestrategy>PreferAntialias</stylestrategy>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>Distance</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Sans Serif</family>
             <pointsize>9</pointsize>
             <stylestrategy>PreferAntialias</stylestrategy>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>Number of Results</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QComboBox" name="businessList">
           <property name="font">
            <font>
             <family>Sans Serif</family>
             <pointsize>9</pointsize>
             <stylestrategy>PreferAntialias</stylestrategy>
             <kerning>false</kerning>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Coffee Shop</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Restaurant</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gas Station</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="colorList">
           <property name="font">
            <font>
             <family>Sans Serif</family>
             <pointsize>9</pointsize>
             <stylestrategy>PreferAntialias</stylestrategy>
             <kerning>false</kerning>
            </font>
           </property>
           <item>
            <property name="text">
             <string>5 Miles</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10 Miles</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>25 Miles</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>50 Miles</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>100 Miles</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numResults">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pubQosButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>9</pointsize>
         <stylestrategy>PreferAntialias</stylestrategy>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="toolTip">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Click to set custom QoS&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>QoS</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createWriterButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>9</pointsize>
         <stylestrategy>PreferAntialias</stylestrategy>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="toolTip">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Click Publish Topic&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Publish</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="pubTextBrowser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>createWriterButton</sender>
   <signal>clicked()</signal>
   <receiver>PubDialog</receiver>
   <slot>onPublishButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pubQosButton</sender>
   <signal>clicked()</signal>
   <receiver>PubDialog</receiver>
   <slot>onWriterQosButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPublishButtonClicked()</slot>
  <slot>onSubscribeButtonClicked()</slot>
  <slot>onWriterQosButtonClicked()</slot>
  <slot>onReaderQosButtonClicked()</slot>
  <slot>onFilterButtonClicked()</slot>
 </slots>
</ui>
